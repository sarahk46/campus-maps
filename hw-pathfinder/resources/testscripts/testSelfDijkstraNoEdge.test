# Find a path with Dijkstra's algorithm between
# one node to itself where there is no edge.

CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A A