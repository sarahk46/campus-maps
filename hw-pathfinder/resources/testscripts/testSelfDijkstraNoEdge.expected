# Find a path with Dijkstra's algorithm between
# one node to itself where there is no edge.

created graph simpleGraph

added node A to simpleGraph

path from A to A:
total cost: 0.000
